low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -214,6 +214,62 @@ def get_platform_sensors_info(self):
                 )
         return response
 
+    def get_nw_ports_info(self):
```

Comment:
I added two comments in jira after looking at the response, i dont think its addressed still
https://jts.seagate.com/browse/EOS-20956?focusedCommentId=2129899&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2129899

Reply:
ohh sure, let me have a look.

Reply:
addressed 👍